Apps/Playground/Android/package.json

```diff
@@ -1,16 +0,0 @@
-{
-  "name": "BabylonNative-android",
-  "version": "1.0.0",
-  "description": "",
-  "main": "index.js",
-  "dependencies": {
-    "jsc-android": "^241213.1.0",
-    "v8-android": "^7.8.2"
-  },
-  "devDependencies": {},
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "author": "",
-  "license": "MIT"
-}
```

Comment:
Hmm, this is used only by Android. I'm not sure it's best to move this to the Apps root.

Reply:
@CedricGuillemet @bghgary happy to align with whatever the group decides on

Reply:
fwiw, these two packages add ~300mb of files. But in the current form we have to install twice/separately for validationtests vs playground so we would go from ~600mb added to ~300mb, but we would be obligating all babylonnative developers to download android dependencies for using this repos apps